DSCResources/MSFT_xDBPackage/MSFT_xDBPackage.psm1

```diff
@@ -109,7 +109,7 @@ function Test-TargetResource
 
     $connectionString = Construct-ConnectionString -sqlServer $SqlServer -credentials $Credentials
 
-    $dbExists = Check-IfDbExists -databaseName $DatabaseName -connectionString $connectionString
+    $dbExists = CheckIfDbExists -connectionString $connectionString -databaseName $DatabaseName 
```

Comment:
Please try to keep these type of changes separate (You could have submitted it before this PR) to make the PRs easier to review.